CatNote.API/DTO/UserRegistrationDTO.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace CatNote.API.DTO
+{
+    public class UserRegistrationDTO
+    {
+        public string? Id { get; set; }
+        public string? UserName { get; set; }
+        public string? Email { get; set; }
```

Comment:
Will be better to remove Email and Password. Later you probably will implement authorization microservice